### PR TITLE
Add a load script which reads a yml file that configures a lilac instance.

### PR DESCRIPTION
**Pull request by nsthorat**

### CODE REVIEW

File: lilac/load.py [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    if not embedding_field:
+      compute_embedding = True
+
+  if compute_embedding:
+    dataset.compute_embedding(embedding_config.embedding, embedding_config.path, task_step_id)
+
+  gc.collect()
+
+
+if __name__ == '__main__':
```
Comment (dsmilkov) [inline lilac/load.py:208]: can you register this as a subcommand in `cli.py`. So users can do `lilac load ...` in addition to `lilac start` and `lilac version`.

**Review by dsmilkov — APPROVED** [review]

File: lilac/load.py [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+import pathlib
+import shutil
+
+import click
+import dask
+import psutil
+import yaml
+from distributed import Client
+
+from lilac.config import Config, EmbeddingConfig, SignalConfig
```
Comment (dsmilkov) [inline lilac/load.py:22]: use relative imports here

File: lilac/config.py [hunk 3]
```diff
@@ -1,63 +1,81 @@
...
-
-
-# Initialize the environment at import time.
-_init_env()
+"""Configurations for a dataset run."""
+from typing import Optional
+
+from pydantic import BaseModel, validator
+
+from lilac.sources.source_registry import resolve_source
```
Comment (dsmilkov) [inline lilac/config.py:6]: use relative import

File: lilac/load.py (hunk 1 above)
Comment (nsthorat) [reply lilac/load.py:208]: done

File: lilac/load.py (hunk 2 above)
Comment (nsthorat) [reply lilac/load.py]: done

File: lilac/config.py (hunk 3 above)
Comment (nsthorat) [reply lilac/config.py]: done

**Pull request merged**